### DISCUSSION TITLE
ESP32-WROOM-32    BLE and DISPLAY rev0

### DISCUSSION THREAD

**Discussion by bartoszwas** [nanoframework/Home#1333] [Q&A] [answered]
Hello everyone,

is there a target firmware that allows using BLE and Display at once? Can't find it on Cloudsmith repo, there's only version with PSRAM, BLE and DISPLAY for rev3 which doesnt work for me...

My chip is ESP32 rev0 (1).

**Comment by Ellerbach** [comment, marked as answer]
There is none as for this device, things starts to get very very very tiny. But you can build your own, clone the nf-interpreter repository and follow the instruction here: https://docs.nanoframework.net/content/building/using-dev-container.html
Use the ESP32 container.
